src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/ConsignmentFields.scala

```diff
@@ -17,6 +17,7 @@ object ConsignmentFields {
   case class ChecksumProgress(filesProcessed: Int)
   case class FFIDProgress(filesProcessed: Int)
   case class FileChecks(antivirusProgress: AntivirusProgress, checksumProgress: ChecksumProgress, ffidProgress: FFIDProgress)
+  case class TransferringBody(bodyid: UUID, name: Option[String], code: Option[String], description: Option[String])
```

Comment:
Do we just need the name at the moment? I'd remove the other fields for now. We know there are some database changes coming, and it will be easier to apply those if the API doesn't refer to fields which might be removed or renamed.

Reply:
Ok, sure!